src/dali_model_instance.cc

```diff
@@ -109,19 +109,33 @@ std::vector<TritonResponse> DaliModelInstance::CreateResponses(
   return responses;
 }
 
+
 ProcessingMeta DaliModelInstance::ProcessRequests(const std::vector<TritonRequest>& requests,
                                                   const std::vector<TritonResponse>& responses) {
   ProcessingMeta ret{};
+
+  TimeRange tr_gi("[DALI BE] GenerateInputs", 0x488FB1);
```

Comment:
```suggestion
  TimeRange tr_proc"[DALI BE] ProcessRequests", kRed1);
  TimeRange tr_gi("[DALI BE] GenerateInputs", 0x488FB1);
```

Reply:
I believe it's not necessary to add one more nested marker here. Almost every line in this function is already covered by nvtx (apart from POD initialization, which should be problematic). Unless I don't see something important?

Reply:
From my point of view it is easier to read one longer timeline and then check the details, but this is just my opinion. I'm not pushing.

Reply:
In the description I've just pasted the screenshot of how it roughly looks right now. Could you take a look? If there is something I should modify/upgrade, please let me know.

Reply:
Looks very nice as it is in the screenshot.